package.xml

```diff
@@ -8,10 +8,17 @@
 
   <license>MIT</license>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <build_depend>mpi_cmake_modules</build_depend>
+
   <depend>libopencv-dev</depend>
-  <depend>pybind11_catkin</depend>
+  <depend>pybind11_vendor</depend>
```

Comment:
here a `buildtool_depend` (or just a `build_depend`?) is better...
I am not sure it matters that much.